sites/public/src/components/shared/FormSummaryDetails.tsx

```diff
@@ -88,21 +88,23 @@ const FormSummaryDetails = ({
     }
   }
 
-  const multiselectQuestionAddress = (extraData?: AllExtraDataTypes[]) => {
+  const multiselectQuestionHelpText = (extraData?: AllExtraDataTypes[]) => {
     if (!extraData) return
-    return extraData.reduce((acc, item) => {
+    const helperText = extraData.reduce((acc, item) => {
       if (item.type === InputType.address && typeof item.value === "object") {
-        acc += `
-          ${item.value.street}${!item.value.street2 && ","}
-          ${item.value.street2 ? `${item.value.street2},` : ""}
-          ${item.value.city},
-          ${item.value.state}
-          ${item.value.zipCode}
-          `
+        acc += `${item.value.street} ${!item.value.street2 ? "," : ""} ${
+          item.value.street2 ? `${item.value.street2},` : ""
+        } ${item.value.city}, ${item.value.state} ${item.value.zipCode}`
       }
 
       return acc
     }, "")
+
+    const name = extraData.find((field) => field.key === "addressHolderName")?.value as string
```

Comment:
we should probably put "addressHolderName" and "addressHolderRelationship" as an enum somewhere since it is use in multiple places